### PR TITLE
Added Button card image components

### PR DESCRIPTION
**Pull request by Mohitk391**

### CODE REVIEW

File: components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+.btn-round {
+    border-radius: 1rem;
+    border-style: none;
+    background-color: var(--bg-danger);
+}
+
+.btn-link {
+    border: none;
+    color: blue;
+    text-decoration: underline;
```
Comment (Ankur9669) [inline components/button/button.css:35]: fix: use css variable here

File: components/card/card.css [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+    height: 15rem;
+    padding: 1rem;
+}
+
+.card {
+    background-color: var(--white);
+    overflow: hidden;
+    object-fit: fill;
+    width: 25rem;
+    border: 1px solid #ccc;
```
Comment (JuvitaSaini09) [inline components/card/card.css]: u could have used variables

**Review by JuvitaSaini09 — APPROVED** [review]
your code looks good to me.UI is also responsive ,good to merge

File: components/button/button.html [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+                    <span class="component_description">btn-outline-icon</span>, 
+                    <span class="component_description">btn-outline-hover</span>, 
+                    <span class="component_description">btn-outline-disabled</span>.
+                </p>
+                <div class="component_wrapper_box">
+                    <div class="button-flex flex">
+                        <button class="btn btn-outline-primary">Primary</button>
+                        <button class="btn btn-outline-round">Rounded </button>
+                        <button class="btn btn-outline-icon"><img src="/assets/arrow_right_alt_icon.svg" alt="->"/></button>
+                        <button class="btn btn-outline-hover">Hover</button>
```
Comment (Ankur9669) [inline components/button/button.html:106]: fix: alt tag should contain some text

File: components/card/card.css [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+.image img {
+    height: 15rem;
+    width: auto;
+    background-repeat: no-repeat;
+}
+
+.image-horizontal img {
+    height: 20rem;
+    width: 15rem;
+    background-repeat: no-repeat;
```
Comment (Ankur9669) [inline components/card/card.css:46]: Suggestion: Using fixes width and height could create problems for responsiveness, you can use max-width and max-height property.

File: components/card/card.html [hunk 5]
```diff
@@ -0,0 +1,275 @@
...
+                    <div class="card">
+                        <div class="image">
+                            <span class="card-badge">Latest</span>
+                            <img src="/assets/card-image.jpg">
+                        </div>
+                        <div class="card-content">
+                            <div class="card-title">Title</div>
+                            <div class="card-sub-title">sub-title</div>
+                            <div class="content">
+                                This a the content body where the description about the product will be present here. It is upto us to choose the length of the description.
```
Comment (Ankur9669) [inline components/card/card.html]: fix: you can wrap the text in p tag

File: components/card/card.html [hunk 6]
```diff
@@ -0,0 +1,275 @@
...
+                            </div>
+                            <div class="footer flex flex-end">
+                                    <div class="card-footer-icons flex">
+                                        <span class="icon-img-container" ><img src="/assets/heart_icon.svg"></span>
+                                        <span  class="icon-img-container"><img src="/assets/share_black_24dp.svg"></span>
+                                        <span  class="icon-img-container" ><img src="/assets/dots_vertical_icon.svg"></span>
+                                    </div>
```
Comment (Ankur9669) [inline components/card/card.html:88]: fix: add alt tags to images

File: components/card/card.html [hunk 7]
```diff
@@ -0,0 +1,275 @@
...
+                        style="width: 800px; height: 514px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
```
Comment (Ankur9669) [inline components/card/card.html:98]: fix: remove inline styling from iframe

File: components/card/card.html [hunk 8]
```diff
@@ -0,0 +1,275 @@
...
+                </div>
+                    
+                <!-- Card with dismiss-->
+                <h3 class="component_heading">Card with dismiss</h3>
+                <div class="component_wrapper_box">
+                    <div class="card">
+                        <div class="image">
+                            <span class="card-badge-icon"><img src="/assets/exit-icon.svg"></span>
+                            <img src="/assets/card-image.jpg">
+                        </div>
```
Comment (Ankur9669) [inline components/card/card.html:109]: fix: alt tag is missing

File: components/card/card.html [hunk 9]
```diff
@@ -0,0 +1,275 @@
...
+                        style="width: 800px; height: 489px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
```
Comment (Ankur9669) [inline components/card/card.html:130]: Same

File: components/card/card.html [hunk 10]
```diff
@@ -0,0 +1,275 @@
...
+                    </iframe>
+                </div>
+
+                <!--Card with text overlay-->
+                <h3 class="component_heading">Card with text overlay</h3>
+                <div class="component_wrapper_box">
+                    <div class="card">
+                        <div class="image flex image-text-overlay">
+                            <img src="/assets/card-image-2.jpg"/>
+                            <div class="card-text-image">
```
Comment (Ankur9669) [inline components/card/card.html:140]: Same

File: components/card/card.html [hunk 11]
```diff
@@ -0,0 +1,275 @@
...
+                        <div class="card-content">
+                            <div class="content">
+                                This a the content body where the description about the product will be present here. It is upto us to choose the length of the description.
+                            </div>
+                            <div class="footer flex flex-end">
+                                    <div class="card-footer-icons flex">
+                                        <span class="icon-img-container" ><img src="/assets/heart_icon.svg"></span>
+                                        <span  class="icon-img-container"><img src="/assets/share_black_24dp.svg"></span>
+                                        <span  class="icon-img-container" ><img src="/assets/dots_vertical_icon.svg"></span>
+                                    </div>
```
Comment (Ankur9669) [inline components/card/card.html:154]: Same alt tag is missing

File: components/card/card.html [hunk 12]
```diff
@@ -0,0 +1,275 @@
...
+                            <div class="footer flex">
+                                    <div class="card-footer-buttons">
+                                        <button class="btn-hover">Previous</button>
+                                        <button class="btn-hover">Next</button>
+                                    </div>
+                                    <div class="card-footer-icons flex">
+                                        <span class="icon-img-container" ><img src="/assets/heart_icon.svg"></span>
+                                        <span  class="icon-img-container"><img src="/assets/share_black_24dp.svg"></span>
+                                        <span  class="icon-img-container" ><img src="/assets/dots_vertical_icon.svg"></span>
+                                    </div>
```
Comment (Ankur9669) [inline components/card/card.html:187]: Same alt tag is missing

File: components/card/card.html [hunk 13]
```diff
@@ -0,0 +1,275 @@
...
+                        <div class="card-sub-title">sub-title</div>
+                        <div class="content">
+                            This a the content body where the description about the product will be present here. It is upto us to choose the length of the description.
+                        </div>
+                        <div class="footer flex flex-end">
+                                <div class="card-footer-icons flex">
+                                    <span class="icon-img-container" ><img src="/assets/heart_icon.svg"></span>
+                                    <span  class="icon-img-container"><img src="/assets/share_black_24dp.svg"></span>
+                                    <span  class="icon-img-container" ><img src="/assets/dots_vertical_icon.svg"></span>
+                                </div>
```
Comment (Ankur9669) [inline components/card/card.html:258]: Same alt tag is missing

File: components/image/image.css [hunk 14]
```diff
@@ -0,0 +1,17 @@
+.reponsive-image {
+    display: flex;
+}
+
+.round-image{
+    width: 20rem;
+    height: 20rem;
+    border-radius: 50%;
```
Comment (Ankur9669) [inline components/image/image.css:8]: Suggestion: If you want to make it responsive you can use max-width and max-height properties

File: components/image/image.html [hunk 15]
```diff
@@ -0,0 +1,98 @@
...
+                </div>
+
+                <div class="component_iframe">
+                    <iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=800&ds=false&dsyoff=0px&dsblur=100px&wc=true&wa=false&pv=0px&ph=0px&ln=true&fl=1&fm=Hack&fs=13.5px&lh=181%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522round-image%2522%253E%253Cimg%2520src%253D%2522...%2522%2520alt%253D%2522...%2522%252F%253E%253C%252Fdiv%253E"
+                        class="image-iframe"
+                        sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+            </div>
```
Comment (Ankur9669) [inline components/image/image.html:94]: fix: alt tags are missing

File: index.html [hunk 16]
```diff
@@ -176,6 +176,94 @@ <h3 class="component_heading">Standar Badges</h3>
...
+                        <span class="component_description">btn-outline-icon</span>, 
+                        <span class="component_description">btn-outline-hover</span>, 
+                        <span class="component_description">btn-outline-disabled</span>.
+                    </p>
+                    <div class="component_wrapper_box">
+                        <div class="button-flex flex">
+                            <button class="btn btn-outline-primary">Primary</button>
+                            <button class="btn btn-outline-round">Rounded </button>
+                            <button class="btn btn-outline-icon"><img src="/assets/arrow_right_alt_icon.svg" alt="->"/></button>
+                            <button class="btn btn-outline-hover">Hover</button>
```
Comment (Ankur9669) [inline index.html:231]: fix: add text in alt tag

File: index.html [hunk 17]
```diff
@@ -176,6 +176,94 @@ <h3 class="component_heading">Standar Badges</h3>
...
+                    The floating action button can be implemented using
+                    <span class="component_description">btn-floating-action</span>,  
+                    and a button with an icon can be implemented using 
+                    <span class="component_description">btn-icon</span>.
+                </p>
+                <div class="component_wrapper_box">
+                    <div class="button-flex flex">
+                        <button class="btn btn-floating-action"><img src="/assets/add_icon.svg" alt="+"/></button>
+                        <button class="btn btn-icon"><span class="btn-icon-text">Login</span><img src="/assets/arrow_forward_icon.svg" alt="->"/></button>
+                    </div>
```
Comment (Ankur9669) [inline index.html:255]: Add text here to

**Review by Ankur9669 — CHANGES_REQUESTED** [review]
The UI and code looks good, there are many minor changes suggested by me which you can do right away

**Review by Ankur9669 — APPROVED** [review]
Code looks good!!!. You can merge it

**Pull request merged**